pkgs/top-level/ocaml-packages.nix

```diff
@@ -566,7 +566,8 @@ let
 
     gapi-ocaml = callPackage ../development/ocaml-modules/gapi-ocaml { };
 
-    gd4o = callPackage ../development/ocaml-modules/gd4o { };
```

Comment:
We should remove the files as well, are there any other packages that use this package?

Reply:
Thought I did. Again my bad. in nixpkgs not to my knowledge